lib/utils/yamlAstParser.js

```diff
@@ -90,7 +90,7 @@ const addNewArrayItem = (ymlFile, pathInYml, newValue) =>
     for (let i = 0; i < pathInYmlArray.length - 1; i++) {
       const propertyName = pathInYmlArray[i];
       const property = currentNode[propertyName];
-      if (_.isUndefined(property) || _.isObject(property)) {
+      if (!property || _.isObject(property)) {
```

Comment:
Here we now change behavior to not throw e.g. on `null` `false` and other falsy values.

Still I think it's acceptable, as as far I understand this code, the error fallback is purely to pick eventual programming error, which I believe is unlikely (we didn't see reports that it happens)

Reply:
Unlikely means we're not 100% sure, right? I think it needs to be monitored to avoid unexpected behavior.

Reply:
While we introduced more permissive logic, there's no chance where scenario (for which it was changed) can occur. 

To me _else_ clause feels as dead code here (but haven't 100% confirmed on that)